### ISSUE TITLE
[BUG] Prompt message flows off screen rather than wrapping

### ISSUE THREAD

**Issue by rthill91** [kazhala/InquirerPy#11]
If the combined length of message and instruction is larger than the terminal width the beginning of the message is lost.

```python3
from InquirerPy import prompt


questions = [{
    "message": "Choose all options that apply.",
    "instruction": ("(Use \u2191\u2193 keys to move between entries and SPACE "
                    "to select or deselect; ENTER will accept the selected values.)"),
    "type": "checkbox",
    "choices": [
        "foo",
        "bar",
        "baz",
    ]
}]

results = prompt(questions)
print(results)
```

```
(.venv) ~$ python sample.py
t apply. (Use ↑↓ keys to move between entries and SPACE to select or deselect; ENTER will accept the selected values.)
❯ ⬡ foo
  ⬡ bar
  ⬡ baz
```

**Comment by kazhala** [comment]
Hi @rthill91 ,

Thanks for reporting. Looks like the cursor at the end of the question line is causing that issue, shouldn't be too hard to fix.

Just FYI, personally not sure if line wrapping is possible at this stage but I'm looking to implement some way to have a dedicated instruction area.

A bit busy lately, things might not move forward as fast.

Thanks,
Kevin

**Comment by rthill91** [comment]
Line wrapping definitely addresses my current needs, but I'm really interested in the dedicated instruction area you mention. I look forward to it in a future update.

Anyway, thanks for the quick turnaround!

**Comment by kazhala** [comment]
Hi @rthill91 ,

Apologies for the delay, the issue is now fixed at master. I'll include this in the next release. I feel like it might be beneficial to have the option to configure the line wrapping behaviour. In the case of not wrapping, I still need to fix the cursor position.

Can't guarantee the time, but I'll try to get this out tmr.

Thanks,
Kevin

**Comment by kazhala** [comment]
Hi @rthill91 ,

I've added the option `wrap_lines` as part of the arguments, it is now released in the v0.2.3 version.

Thanks,
Kevin
